php-classes/Slate/CBL/Demonstrations/DemonstrationSkill.php

```diff
@@ -35,9 +35,10 @@ class DemonstrationSkill extends \VersionedRecord
             'unsigned' => true,
             'default' => null
         ],
-        'Override' => [
-            'type' => 'boolean',
-            'default' => false
+        'EvidenceWeight' => [
+            'type' => 'tinyint',
+            'unsigned' => true,
+            'default' => 1
         ]
```

Comment:
- [ ] double-check the current codebase for any other instances of `Override` that might need to be removed/ported